administration/src/util/helper.ts

```diff
@@ -1,8 +1,9 @@
 import XRegExp from 'xregexp'
 
-export const isProductionEnvironment = (): boolean => process.env.REACT_APP_IS_PRODUCTION === 'true'
-export const isDevMode = (): boolean => window.location.hostname === 'localhost'
-export const isStagingMode = (): boolean => !!window.location.hostname.match(/staging./)
+export const isStagingEnvironment = (): boolean => !!window.location.hostname.match(/staging./)
+export const isProductionEnvironment = (): boolean =>
+  process.env.REACT_APP_IS_PRODUCTION === 'true' && !isStagingEnvironment()
+export const isDevEnvironment = (): boolean => window.location.hostname === 'localhost'
```

Comment:
:thinking: I think a better solution would be to replace the boolean `REACT_APP_IS_PRODUCTION` with a string `REACT_APP_ENVIRONMENT` which can be set to `production`, `staging` or `development`. Otherwise using the ip of localhost might not work for example.

:upside_down_face: I think the naming of the environment variables is not really great as the `REACT_APP` is very redundant (and does not even follow our other naming).

Reply:
I'm not sure if environment variables without `REACT_APP_` will be recognized without additional implementation. At least for `create-react-app` projects it was the case. 

As i mentioned in the description. Setting a variable to `staging` won't help us, since we build the packages for staging and just promote them. Thats why we can only distinguish in case of build variables between development or not. If we want to change that, we would have to build different packages (staging/production) with i would like to avoid.

Reply:
well the question is then how we distinguish between staging and production also via the hostname.
But we could turn the variable to `_IS_DEVELOPMENT` and then check for false if the hostname contains staging. Then also for other ip addresses than localhost development will be used

Reply:
I don't get that. We can keep the current checks for staging and production and just alter the check for development?